analysis.go

```diff
@@ -0,0 +1,53 @@
+package goInspector
```

Comment:
Can you use_all lowercase package name like `go_inspector` ?

Reply:
It is a good idea!